pcsx2/DEV9/ATA/Commands/ATA_SCE.cpp

```diff
@@ -49,7 +59,33 @@ void ATA::SCE_IDENTIFY_DRIVE()
 {
 	PreCmd();
 
-	//HDD_IdentifyDevice(); //Maybe?
+	// fill sceSec response with default data
+	memcpy(sceSec, "Sony Computer Entertainment Inc.", 32);
+	memcpy(sceSec + 0x20, "SCPH-20401", 10);
+	memcpy(sceSec + 0x30, "  40", 4);
+
+	sceSec[0x40] = 0; // 4-byte HDD internal SCE serial, always starts with 0x02
+	sceSec[0x41] = 0;
+	sceSec[0x42] = 0;
+	sceSec[0x43] = 0x02;
+
+	sceSec[0x46] = 0x1a; // unknown 8-bytes
+	sceSec[0x47] = 0x01;
+	sceSec[0x48] = 0x02;
+	sceSec[0x49] = 0x20;
+	sceSec[0x4c] = 0x01;
+	sceSec[0x4d] = 0x03;
+	sceSec[0x4e] = 0x11;
+	sceSec[0x4f] = 0x01;
+	// 0x50 - 0x80 is a random unique block of data
+
+	// if exists .hddid file along with HDD image, use its content for sceSec response
+	std::string hddid_file(Path::ReplaceExtension(EmuConfig.DEV9.HddFile.c_str(), "hddid"));
+	auto fp = FileSystem::OpenManagedCFile(hddid_file.c_str(), "rb");
+	if (fp && FileSystem::FSize64(fp.get()) >= 128)
+		std::fread(sceSec, 1, 128, fp.get());
```

Comment:
Error checking?

Better to just read the whole thing and see how many bytes it got.

Reply:
did it the same way as here https://github.com/PCSX2/pcsx2/blob/master/pcsx2/CDVD/CDVD.cpp#L124